interface/patient_file/summary/demographics.php

```diff
@@ -1673,7 +1673,7 @@ function setMyPatient() {
                             $dispmin  = substr($row['pc_startTime'], 3, 2);
                             if ($disphour >= 12) {
```

Comment:
In this script, it looks like may also need to deal with the $dispampm variable (should it not be either am or pm when using 24H time?).

Reply:
It still sets it based on whether it is greater than 12. I could remove it but I wasn't sure  if it was better to leave it so it says 13:45 pm or to remove it altogether.

Reply:
remove the am/pm when using 24H time